File: Support-EJ1/UploadBox/WebForms/FileSelect/FileSelect/SaveFile.ashx.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Web;

namespace FileSelect
{
    /// <summary>
    /// Summary description for SaveFile
    /// </summary>
    public class SaveFile : IHttpHandler
    {

        public void ProcessRequest(HttpContext context)
        {
            string targetFolder = HttpContext.Current.Server.MapPath("uploadfiles");

            if (!Directory.Exists(targetFolder))

            {

                Directory.CreateDirectory(targetFolder);

            }

            HttpRequest request = context.Request;

            HttpFileCollection uploadedFiles = context.Request.Files;

            if (uploadedFiles != null && uploadedFiles.Count > 0)

            {

                for (int i = 0; i < uploadedFiles.Count; i++)

                {

                    if (uploadedFiles[i].FileName != null && uploadedFiles[i].FileName != "")

                    {

                        string fileName = uploadedFiles[i].FileName;

                        int index = fileName.LastIndexOf("\\");

                        if (index > -1)

                        {

                            fileName = fileName.Substring(index + 1);

                        }

                        uploadedFiles[i].SaveAs(targetFolder + "\\" + fileName);

                    }

                }

            }
        }

        public bool IsReusable
        {
            get
            {
                return false;
            }
        }
    }
}

File: EJ1-Components-exmples/UploadBox/WebForms/UploadBox_VB/SaveFile.ashx.vb
Imports System.IO
Imports System.Web
Imports System.Web.Services

Public Class SaveFile
    Implements System.Web.IHttpHandler

    Sub ProcessRequest(ByVal context As HttpContext) Implements IHttpHandler.ProcessRequest

        Dim targetFolder As String = HttpContext.Current.Server.MapPath("tempFiles")
        If Not Directory.Exists(targetFolder) Then
            Directory.CreateDirectory(targetFolder)
        End If

        Dim request As HttpRequest = context.Request
        Dim uploadedFiles As HttpFileCollection = context.Request.Files
        If uploadedFiles IsNot Nothing AndAlso uploadedFiles.Count > 0 Then

            Dim i As Integer = 0

            While i < uploadedFiles.Count
                Dim fileName As String = uploadedFiles(i).FileName
                Dim indx As Integer = fileName.LastIndexOf("\")
                If indx > -1 Then
                    fileName = fileName.Substring(indx + 1)
                End If

                'fileName = "TEMP-" & DateTime.Now.ToString("MMddyyyy") & "_" & Replace(LCase(Strings.Trim(fileName)), " ", "_") 
                uploadedFiles(i).SaveAs(targetFolder & "\" & LCase(fileName))

                i += 1


            End While

        Else
        End If

    End Sub

    ReadOnly Property IsReusable() As Boolean Implements IHttpHandler.IsReusable
        Get
            Return False
        End Get
    End Property

End Class